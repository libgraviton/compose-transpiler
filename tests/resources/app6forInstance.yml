components:
  redis2:
    name: redis
    instances: 3
    forInstance2:
      mixins:
        nomanager:

    mixins:
      manager:
