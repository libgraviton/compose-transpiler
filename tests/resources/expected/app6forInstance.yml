version: 99
services:
  redis:
    image: 'redis:latest'
    environment:
      ENV_NAME: "${REDIS_NAME}"
    deploy:
      placement:
        constraints:
          - 'node.role == manager'
  redis2:
    image: 'redis:latest'
    environment:
      ENV_NAME: "${REDIS2_NAME}"
    deploy:
      placement:
        constraints:
          - 'node.role != manager'
  redis3:
    image: 'redis:latest'
    environment:
      ENV_NAME: "${REDIS3_NAME}"
    deploy:
      placement:
        constraints:
          - 'node.role == manager'
networks:
  internal:
    external: false
